UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -513,7 +516,15 @@
 !endif
 !endif
 
+
 [PcdsPatchableInModule.X64]
+  #
+  # The following parameters are set by Library/PlatformHookLib
+  #
+  gEfiMdeModulePkgTokenSpaceGuid.PcdSerialUseMmio|FALSE
+  gEfiMdeModulePkgTokenSpaceGuid.PcdSerialRegisterBase|0x3F8
+  gEfiMdeModulePkgTokenSpaceGuid.PcdSerialBaudRate|$(BAUD_RATE)
+  gEfiMdeModulePkgTokenSpaceGuid.PcdSerialRegisterStride|1

```

Comment:
These PCDs are listed in line 542 ~ 545.

Reply:
PCDs in 524-527 are for PcdsPatchableInModule.X64.
PCDs in  542-545 are for PcdsPatchableInModule.common

Reply:
see if we can clean .X64 duplicate lines.

Reply:
resolved